data/json/recipes/food/brewing.json

```diff
@@ -0,0 +1,236 @@
+        "components": [
+            [ [ "water", 15 ], [ "water_clean", 15 ] ],
+            [ [ "barley", 3 ], [ "buckwheat", 3 ], [ "oats", 3 ] ],
+            [ [ "hops", 1 ] ],
+            [ [ "yeast", 1 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "brew_mead",
+        "result_mult" : 3,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 3,
+        "time": 80000,
+        "batch_time_factors": [50, 4],
+        "autolearn": true,
+        "book_learn": [ ["brewing_cookbook", 2] ],
+        "components": [
+            [ [ "water", 3 ], [ "water_clean", 3 ] ],
+            [ [ "honey_bottled", 10 ], [ "honeycomb", 2 ], [ "honey_glassed", 10 ] ],
+            [ [ "hops", 1 ] ],
+            [ [ "yeast", 1 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "brew_dandelion_wine",
+        "result_mult" : 3,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 3,
+        "time": 80000,
+        "batch_time_factors": [50, 4],
+        "book_learn": [
+            ["brewing_cookbook", 3], ["family_cookbook", 2], ["survival_book", 3],
+            ["textbook_survival", 3], ["manual_survival", 3]
+        ],
+        "qualities":[
+            { "id":"CUT","level":1 },
+            { "id": "COOK", "level": 2 }
+        ],
+        "components": [
+            [ [ "water", 3 ], [ "water_clean", 3 ] ],
+            [ [ "raw_dandelion", 10 ] ],
+            [ [ "sugar", 10 ], [ "honeycomb", 1 ], [ "honey_bottled", 2 ], [ "honey_glassed", 2 ] ],
+            [ [ "yeast", 1 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "brew_pine_wine",
+        "result_mult" : 3,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 3,
+        "time": 80000,
+        "batch_time_factors": [50, 4],
+        "reversible": false,
+        "autolearn": false,
+        "book_learn": [ ["survival_book", 3] , ["textbook_survival", 3] , ["manual_survival", 3] ],
+        "qualities":[
+            { "id":"CUT","level":1 },
+            { "id": "COOK", "level": 2 }
+        ],
+        "components": [
+            [ [ "water", 3 ], [ "water_clean", 3 ] ],
+            [ [ "pine_bough", 5 ] ],
+            [ [ "sugar", 10 ], [ "honeycomb", 1], [ "honey_bottled", 2 ], [ "honey_glassed", 2 ] ],
+            [ [ "yeast", 1 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "brew_vinegar",
+        "result_mult" : 7,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "cooking",
+        "difficulty": 3,
+        "time": 8000,
+        "batch_time_factors": [50, 4],
+        "reversible": false,
+        "autolearn": true,
+        "components": [
+            [ [ "water", 7 ], [ "water_clean", 7 ] ],
```

Comment:
A separate PR could allow milk and wine to rot to vinegar and curdled milk with the latter then itself promptly rotting